fix(curriculum): Updated description of step 119 of JavaScript Algorithms and Data Structures Role Playing Game

developwithabhinav opened the pull request:
Checklist:

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

Closes #53601

<!-- Feel free to add any additional description of changes below this line -->

developwithabhinav committed:
Updated the description and instructions of Step 78 of Platformer game

Updated the Instructions for step 78 of the platformer game as it was unclear for some learners. Instead of using the *if* statement in the line:

`Inside that array, add a boolean expression that checks if the player's y position plus the player's height is less than or equal to the platform's y position.`

and changed it to whether so, it's not confusing for learners that they have to use it if statement or not

developwithabhinav committed:
Merge branch 'freeCodeCamp:main' into main

developwithabhinav committed:
Updated description of step 119 of Role Playing Game

Updated the description of step 119 of the "JavaScript Algorithms and Data Structures" role-playing game and made it clearer for the learners that they must use compound assignment.

jdwilkin4 commented:
Hi @developwithabhinav ! 

Thank you for your interest in contributing.

In the future though, it is best to wait until the issue has been triaged before you begin working on it.
Because not all created issues, will be worked on some will be closed without further action.

I'll up the issue you are referencing for help wanted

jdwilkin4 commented on curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8e41e2f190c58404dd46e.md:
```diff
@@ -6,8 +6,7 @@ dashedName: step-118
 # --description--
-
-On a new line, add the string `" You attack it with your <weapon>."` to the `text` value, replacing `<weapon>` with the player's current weapon. Additionally, remember that this line of text starts with a space so it will properly display. 
+On a new line, append the string `" You attack it with your <CURRENT_WEAPON_NAME>."` to the current value of `text`, and replace `<CURRENT_WEAPON_NAME>` with the player's current weapon. Additionally, remember that this new string starts with a space to display it properly.
```
Let's go with these changes mentioned in the issue here

https://github.com/freeCodeCamp/freeCodeCamp/issues/53601#issuecomment-1933252876

developwithabhinav replied:
I did it, please check 
Also, I will wait for someone's review to new issues before making any change.

Thank you very much

developwithabhinav committed:
fix(curriculum): Updated description of step 118 of JavaScript Algorithms and Data Structures Role Playing Game

Updated the description of step 118 of the "JavaScript Algorithms and Data Structures" role-playing game and made it clearer for the learners that they must use compound assignment.

jdwilkin4 commented on curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8e41e2f190c58404dd46e.md:
```diff
@@ -6,11 +6,9 @@ dashedName: step-118
+On a new line, use the addition assignment operator(`+=`), to add the string `" You attack it with your <weapon>."` to the `text` value, replacing `<weapon>` with the player's current weapon. Additionally, remember that this line of text starts with a space so it will properly display. 
 
 # --hints--
-
```
we don't need to remove this line.
Please revert this back to the way it was 👍

jdwilkin4 reviewed (changes requested):
Everything looks.
You just need to resolve that formatting change 👍

gikf commented on curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8e41e2f190c58404dd46e.md:
```diff
@@ -6,11 +6,9 @@ dashedName: step-118
 ---
 
 # --description--
```
```suggestion
# --description--

```

developwithabhinav committed:
Updated the Pattern of the description

Update the pattern of the description of the step 118 of the role playing game of Javascript Algorithms and Data Structures
